Assignment-3/Area.java

```diff
@@ -0,0 +1,116 @@
+import java.util.Scanner;
+class Shapes {
+	private double result;
+	/**
+	 * This function gives area of triangle
+	 * @param width > 0
+	 * @param height > 0
+	 * @return area of triangle in double
+	 * @throws ArithmeticException, if user has entered wrong values
+	 */
+	public double traingle(double width,double height) throws ArithmeticException {
+		double base_traingle=width;
+		double height_traingle=height;
+		double area;
+		area=0.5*base_traingle*height_traingle;   
+		return area;
+	}
+	/**

```

Comment:
@meta-aakanksha-raj  Always provide a line break between function definitions.
Please do that for all the functions.